qiskit/assembler/assemble_circuits.py

```diff
@@ -91,11 +91,11 @@ def assemble_circuits(circuits, run_config, qobj_id, qobj_header):
             qargs = op_context[1]
             cargs = op_context[2]
             if qargs:
-                qubit_indices = [qubit_labels.index([qubit[0].name, qubit[1]])
+                qubit_indices = [qubit_labels.index([qubit.register.name, qubit.index])
```

Comment:
Could this just be `qubit_labels.index(qubit)`?

Reply:
Perhaps you're leaving that to a future PR but without it you don't get the benefit of not having to touch almost all the same pieces of code again when it is implemented.

Reply:
No.. it is not equivalente, because `qubit_labels`'s keys are things like `['c1', 0]`. I do see that there is room for optimization... but I would prefer to do that in a different PR.